serveradmin/servershell/static/js/servershell/command.js

```diff
@@ -67,35 +67,42 @@ servershell.update_attribute = function(object_id, attribute_id, value) {
     }
 
     if (attribute.multi) {
-        // Get changes to current (saved) state
+        // Determine changes to add and remove for to_commit.
         let to_add = value.filter(v => !object[attribute_id].includes(v));
         let to_remove = object[attribute_id].filter(v => !value.includes(v));
 
-        changes[object_id][attribute_id] = {
-            'action': 'multi',
-            'add': to_add,
-            'remove': to_remove,
-        }
-
+        // It might be that there is nothing more to change for this attribute
         if (to_add.length === 0 && to_remove.length === 0) {
-            delete changes[object_id][attribute_id];
+            if (changes[object_id].hasOwnProperty(attribute_id)) {
```

Comment:
I tried this in my local browser and it throws an exception. I tried this again and attached the screenshot - am I holding it wrong ?

![Screenshot from 2021-02-17 07-58-05](https://user-images.githubusercontent.com/3993306/108167399-ec60d700-70f5-11eb-8a2e-229e55704e07.png)

Reply:
That's correct, that that does not work, but that's also not the case what we have here. You should have tested:
```
let changes = {}
changes['the_object_id'] = {}
delete changes['the_object_id']['the_attribute_id']
```
That should work. That your's not working is about that it can't access 'the_attribute_id' because 'the_object_id' would already result in `undefined` and therefore it can't access anything on `undefined`.

Reply:
But it's also not important to change, you can also ignore it.